src/JBrowse/Store/SeqFeature/VCFTabix/LazyFeature.js

```diff
@@ -95,7 +95,7 @@ var Feature = Util.fastDeclare(
         }
 
         // add a toString to it that serializes it to JSON without its metadata
-        bySample.toString = this._stringifySample;
+        //bySample.toString = this._stringifySample;
```

Comment:
why remove the stringification code?

Reply:
The feature.get('genotypes') would also contain a toString element that you have to work around. The toString would appear in Object.keys(feature.get('genotypes')) and such and seems unnecessary?